### PR TITLE
Update yupana to use new report structure

### LINKED ISSUE #74: Yupana should process new QPC insights report format
## Feature description:
Yupana should process new QPC insights report format
See new format details here: https://github.com/quipucords/quipucords/issues/1815

### PR DESCRIPTION
**Pull request by abaiken**
Closes #74

### CODE REVIEW

File: yupana/processor/kafka_msg_handler.py [hunk 1]
```diff
@@ -240,69 +240,95 @@ def verify_report_details(account_number, deployments_report):
...
-    fingerprints = deployments_report['system_fingerprints']
-    report_id = deployments_report['report_platform_id']
+    hosts = insights_report['hosts']
+    report_id = insights_report['report_platform_id']
 
-    prefix = 'VALIDATE FINGERPRINTS'
-    valid_fingerprints = []
-    invalid_fingerprints = []
-    for fingerprint in fingerprints:
+    prefix = 'VALIDATE hosts'
```
Comment (myersCody) [inline yupana/processor/kafka_msg_handler.py]: All of the prefixes are written in all caps.

File: yupana/processor/kafka_msg_handler.py [hunk 2]
```diff
@@ -391,41 +417,41 @@ def upload_to_host_inventory(account_number, report_id, fingerprints):
...
+        for failed_info in failed_hosts:
             LOG.error(format_message(
                 prefix,
-                'Host inventory returned %s for %s.  Error: %s.  Fingerprint: %s' % (
+                'Host inventory returned %s for %s.  Error: %s.  host: %s' % (
                     failed_info.get('status_code'),
                     failed_info.get('display_name'),
                     failed_info.get('error'),
-                    failed_info.get('fingerprint')),
+                    failed_info.get('host')),
```
Comment (myersCody) [inline yupana/processor/kafka_msg_handler.py]: Do we want to log the host_id which is the system_platform_id when we have a failure? We didn't have that option before, but now we do.

Comment (abaiken) [reply yupana/processor/kafka_msg_handler.py]: great catch 😄 Done!

File: yupana/processor/kafka_msg_handler.py (hunk 1 above)
Comment (abaiken) [reply yupana/processor/kafka_msg_handler.py]: 😬You're right!

**Comment by chambridge** [comment]
👏

**Pull request merged**